src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -246,6 +246,13 @@ def EstablishPASESessionIP(self, ipaddr, setupPinCode, nodeid):
                 self.devCtrl, ipaddr, setupPinCode, nodeid)
         )
 
+    def StopPairing(self, nodeid):
```

Comment:
Should this be called `StopCommissioning` ?

Reply:
I think it's named poorly, but it's named to match the underlying C++ API, which is already in use in a lot of platforms.

Reply:
Going to leave it for now, so the APIs aren't mismatched.